iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/unique_ptr.inl

```diff
@@ -47,8 +47,10 @@ inline unique_ptr<T>& unique_ptr<T>::operator=(unique_ptr&& rhs) noexcept
 
 template <typename T>
 inline unique_ptr<T>::unique_ptr(unique_ptr&& rhs) noexcept
+    : m_ptr{rhs.m_ptr}
+    , m_deleter{rhs.m_deleter}
```

Comment:
Why not move for m_deleter?

Reply:
Oh I fixed it but accidentally moved the deleter in the copy constructur ... Whoopsie, now it is correct.